src/models/Advertisement.ts

```diff
@@ -11,7 +11,7 @@ export interface InterfaceAdvertisement {
   _id: Types.ObjectId;
   orgId: string;
   name: string;
-  link: string;
+  mediaUrl: string | undefined | null;
```

Comment:
`mediaUrl` is required in mongoose and graphql schema, why would you put the typescript type to be nullable?